dash_core_components/package.json

```diff
@@ -57,6 +57,7 @@
     "babel-eslint": "^10.0.1",
     "babel-jest": "^24.5.0",
     "babel-loader": "^8.0.5",
+    "check-prop-types": "^1.1.2",
```

Comment:
What's the reason for using `check-prop-types` instead of `prop-types`?

Reply:
the default prop-types check swallows the error, instead print a warn message in console.
the check-* one is basically a wrapper to throw the error back